### ISSUE TITLE
`SPC u p` pastes 4 times with indentation

### ISSUE THREAD

**Issue by xuhdev** [syl20bnr/spacemacs#13241]
#### Description :octocat:
`SPC u p` pastes 4 times with indentation, but we expect it to paste one time without indentation -- #11340 seems to have come back.

#### Reproduction guide :beetle:
- Start Emacs
- Open a file
- In normal mode, run `yy`, then `SPC u p`.

*Observed behaviour:* :eyes: :broken_heart:
Pasted 4 times with indentation.

*Expected behaviour:* :heart: :smile:
Pasted once without indentation.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3.50
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 7b28e62fb02a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: ivy
- Layers:
```elisp
(ivy auto-completion better-defaults bibtex
     (c-c++ :variables c-c++-default-mode-for-headers 'c++-mode c-c++-enable-clang-format-on-save t)
     cmake csv dash debug docker emacs-lisp git github gnus gpu graphviz html
     (latex :variables latex-enable-folding t)
     lsp markdown php
     (python :variables python-backend 'lsp python-test-runner
             '(pytest nose))
     (ranger :variables ranger-show-preview t ranger-cleanup-on-disable t ranger-show-literal nil)
     restructuredtext search-engine shell shell-scripts spell-checking sphinx syntax-checking
     (treemacs :variables treemacs-use-follow-mode 'tag treemacs-use-filewatch-mode t treemacs-use-git-mode 'extended)
     vagrant
     (version-control :variables version-control-diff-side 'left)
     (xclipboard :variables xclipboard-enable-cliphist t)
     yaml)
```
- System configuration features: XPM JPEG TIFF GIF PNG SOUND DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS

**Comment by duianto** [comment]
`SPC u p` didn't paste without indentation after the emacs-evil/evil package fixed the broken `C-u p` error that you reported here:
Paste with universal-argument reports error https://github.com/emacs-evil/evil/issues/1199

A Spacemacs PR was opened with a possible solution:
Fix bug: disable paste auto-indent #13161

But bet4it noticed that it could be fixed upstream in evil:
https://github.com/syl20bnr/spacemacs/pull/13161#issuecomment-575167590

And opened this upstream PR, that's been merged:
Change the way to get universal argument https://github.com/emacs-evil/evil/pull/1259

My comment in the Spacemacs PR demonstrates how `SPC u p` works now with the latest evil package update.

If your not seeing the same behavior, check if you have some user configuration that was fixing the previous pasting issue.

**Comment by xuhdev** [comment]
@duianto I'm a bit confused. Are you suggesting that, in the current develop branch, `SPC u p` should work as pasting once without indentation? I'm on hybrid mode and `C-u C-y` in insert mode works as expected.

Running `C-h p` shows:

```
p runs the command evil-paste-after (found in evil-normal-state-map), which is an interactive
compiled Lisp function in ‘~/.dotfiles/emacs.d/elpa/develop/evil-20200203.2006/evil-commands.el’.

It is bound to p.

(evil-paste-after COUNT &optional REGISTER YANK-HANDLER)

:around advice: ‘spacemacs//yank-indent-region’

Pastes the latest yanked text behind point.
The return value is the yanked text.
```

**Comment by duianto** [comment]
>Are you suggesting that, in the current develop branch, SPC u p should work as pasting once without indentation?

That's my understanding of how it used to work before the evil package fixed the universal argument pasting issue:
Paste with universal-argument reports error emacs-evil/evil#1199

Since then, the universal argument has been updated in the evil package:
Change the way to get universal argument emacs-evil/evil#1259

And now `SPC u p` pastes once without indentation.

When I asked how this PR should be handled:
Fix bug: disable paste auto-indent #13161

syl20bnr said:
>C-u p should past only one time without indentation.

---

There hasn't been a mention of how pasting in Emacs (and hybrid insert state) `C-u C-y` should be handled. If it works as expected, then Spacemacs might not be modifying it. That's probably what Emacs editing style (and hybrid insert state) users expect.

This PR mentions `C-u C-y` pasting:
Check for universal argument before paste #11340

**Comment by xuhdev** [comment]
I'm not sure how, but the latest version fixed the issue. I'll report again if it resurfaces. Thanks for the response!
